libraries/AC_AttitudeControl/AC_AttitudeControl_Multi.cpp

```diff
@@ -232,6 +232,20 @@ const AP_Param::GroupInfo AC_AttitudeControl_Multi::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("THR_MIX_MAN", 6, AC_AttitudeControl_Multi, _thr_mix_man, AC_ATTITUDE_CONTROL_MAN_DEFAULT),
 
+    // @Param: RP_PD_BOOST
+    // @DisplayName: PD boost
+    // @Description: PD boost
```

Comment:
Can you add more instructions please?  We should explicitly spell out that PD_Boost of 1 is effectively off/has no effect.  it is common in our params that we set 0 or -1 for off/no effect.

Reply:
Fixed